test/integration/targets/known_hosts/tasks/main.yml

```diff
@@ -170,6 +170,95 @@
     - 'result.diff.before == result.diff.after'
     - 'known_hosts_v3.stdout == known_hosts_v4.stdout'
 
+# test addition as hashed_host
+
+- name: add a new hashed host
+  known_hosts:
+    name: example.org
+    key: "{{ example_org_rsa_key }}"
+    state: present
+    path: "{{output_dir|expanduser}}/known_hosts"
+    hash_host: yes
+  register: result
+
+- name: get the file content
+  shell: cat "{{output_dir|expanduser}}/known_hosts"
```

Comment:
`command` module should be used instead of `shell`

Reply:
`expanduser` isn't needed here if the double quote are moved (`command: "cat {{output_dir}}/known_hosts"`)